charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -72,6 +72,9 @@ spec:
               enable_lazy_spilo_upgrade:
                 type: boolean
                 default: false
+              enable_cross_namespace_secret:
```

Comment:
please, follow alphabetical sorting. Makes it easier later to find stuff. And I would move it under the `kubernetes` section, as it's about K8s secrets